operators/advanced_network/Dockerfile

```diff
@@ -24,13 +24,20 @@ ARG BUILDARCH
 RUN echo "Using build architecture ${BUILDARCH}"
 RUN echo "Using architecture $TARGETPLATFORM"
 
-RUN apt update && apt install -y python3-pyelftools ninja-build meson 
+RUN apt update && apt install -y python3-pyelftools ninja-build meson libyaml-cpp-dev
 RUN pip3 install scipy loguru attrs
 WORKDIR /opt
 
-RUN    eval `dpkg-architecture` \
-        && wget https://www.mellanox.com/downloads/DOCA/DOCA_v2.7.0/host/doca-host_2.7.0-204000-24.04-ubuntu2204_${BUILDARCH}.deb -O doca-host.deb \
-        && ls -lh && apt install ./doca-host.deb \
-        && apt update \
-        && apt install -y doca-all \
-        && apt install -y doca-gpu doca-gpu-dev
+ARG DEBIAN_FRONTEND=noninteractive
+ARG DOCA_REPO_LINK=https://linux.mellanox.com/public/repo/doca/2.8.0/ubuntu22.04/x86_64
```

Comment:
This looks like it removes BUILDARCH

Reply:
because BUILDARCH says amd64 but the link is at x86_64

Reply:
Should we parameterize on that so we don't need two dockerfiles?